packages/block-library/src/gallery/index.php

```diff
@@ -130,7 +116,47 @@ function block_core_gallery_render( $attributes, $content ) {
 			'context' => 'block-supports',
 		)
 	);
-	return (string) $processed_content;
+
+	$updated_content = $processed_content->get_updated_html();
+
+	/*
+	 * Randomize the order of image blocks. Ideally we should shuffle
+	 * the `$parsed_block['innerBlocks']` via the `render_block_data` hook.
+	 * However, this hook doesn't apply inner block updates when blocks are
+	 * nested.
+	 * @todo: In the future, if this hook supports updating innerBlocks in
+	 * nested blocks, it should be refactored.
+	 *
+	 * @see: https://github.com/WordPress/gutenberg/pull/58733
+	 */
+	if ( empty( $attributes['randomOrder'] ) ) {
+		return $updated_content;
+	}
+
+	// Matches figure elements with class `wp-block-image`
```

Comment:
```suggestion
	// This pattern matches figure elements with the `wp-block-image` class to
	// avoid the gallery's wrapping `figure` element and extract images only.
```

The current comment misses the trailing `.` again so triggers a linting error. I think we could also tweak the comment to reflect a little more of the intent of the regex. What do you think?

Reply:
This is definitely easier to understand 👍